30-performance/TPC-DS Runner/utils/run.py

```diff
@@ -0,0 +1,36 @@
+from utils.general import setup_files
+from utils.databricks_client import DatabricksClient
+
+
+def run(spark, dbutils, constants):
+    # Step 0: drop write schema if exists
+    spark.sql(
+        f"drop schema if exists {constants.catalog_name}.{constants.schema_name} cascade"
+    )
+
+    # Step 1: write init script, jar, and beaker whl to DBFS
+    setup_files(
+        dbutils,
+        constants.jar_path,
+        constants.init_script_path,
+        constants.beaker_whl_path,
+    )
+
+    # Step 2: create the client
+    client = DatabricksClient(constants)
+
+    # Step 3: create a warehouse to benchmark against
+    warehouse_id = client.create_warehouse().id
+    constants.warehouse_id = warehouse_id
+
+    # Step 4: create a job to create TPCDS data and queries at a given location and runs benchmarks
```

Comment:
I saw from the running job that the data (and maybe queries) are being created at 

dbfs:/FileStore

Why here? /FileStore is a special location for accessibility from outside... was there a specific reason to put things under there?

Reply:
Good point. All data/queries live in that directory. I changed it to `dbfs:/Benchmarking/TPCDS/...`. Thanks for the spot!